app/lib/boolean-phrase-parser.js

```diff
@@ -0,0 +1,208 @@
+(function (_) {
```

Comment:
This is a bit of code evolution issue. First, each file was in a separate IIFE which got all the dependencies in the call but then I needed specific load order between the functions and a header so I just put all the dependencies in the header and now all of the files are wrapped in a single IIFE with the needed dependencies. The IIFE in each file is only needed to create scope for local variables and not for dependencies. I need to remove the dependency for EJS from all the IIFE in the files.